ocrd_utils/ocrd_utils/logging.py

```diff
@@ -88,23 +92,25 @@ def getLogger(*args, **kwargs):
     Wrapper around ``logging.getLogger`` that respects `overrideLogLevel <#setOverrideLogLevel>`_.
     """
     logger = logging.getLogger(*args, **kwargs)
-    if _overrideLogLevel is not None:
+    if _overrideLogLevel:
         logger.setLevel(logging.NOTSET)
+    with open('/tmp/debug.log', 'a') as f:
+        f.write('getLogger(%s) level=%s\n' % (args[0], logger.level))
     return logger
 
 # Default logging config
 
-def initLogging():
+def initLogging(reinit=False):
     """
     Reset root logger, read logging configuration if exists, otherwise use basicConfig
     """
+    global _initialized_flag
+    if _initialized_flag and not reinit:
+        raise Exception("initLogging called multiple times")
```

Comment:
Because I wanted to catch multiple calls of initLogging. And *if* it is called multiple times, there is probably the intention of re-initialization, which a no-op early return would defeat. But since that shouldn't happen anymore now, the `reinit` mechanism can go AFAICS and instead of an excecption, a stern warning might be more sensible.

Reply:
Now just a warning with source of call, no exception, no return.